dev-setup/gardenadm/machines/machine.yaml

```diff
@@ -44,7 +44,7 @@ spec:
       automountServiceAccountToken: false
       hostAliases:
         - hostnames:
-          - api.root.garden.internal.gardenadm.local
+          - api.root.garden.external.local.gardener.cloud
```

Comment:
I don't think we need this `external`. This suggests that there might be something else (like `internal`), but this isn't the case for ASCs.